modules/core/src/v2/coins/bsv.ts

```diff
@@ -47,31 +48,13 @@ export class Bsv extends Bch {
     return common.Environments[this.bitgo.getEnv()].bsvExplorerBaseUrl + url;
   }
 
-  getAddressInfoFromExplorer(addressBase58: string): Bluebird<any> {
-    const self = this;
-    return co(function* getAddressInfoFromExplorer() {
-      // TODO BG-9989: Update this method with the correct API route and parsing once we have one
-      const addrInfo = yield request.get(self.recoveryBlockchainExplorerUrl(`/addr/${addressBase58}`)).result();
-
-      addrInfo.txCount = addrInfo.txApperances;
-      addrInfo.totalBalance = addrInfo.balanceSat;
-
-      return addrInfo;
-    }).call(this);
+  async getAddressInfoFromExplorer(addressBase58: string, apiKey?: string): Bluebird<any> {
```

Comment:
I think we can narrow the return type here to `Bluebird<AddressInfo>` and `Bluebird<UnspentInfo[]>`, since those are the return types for the function's you're overriding.

Reply:
ah yes, thanks for catching!